cmd/crates/soroban-spec-typescript/src/project_template/src/assembled-tx.ts

```diff
@@ -297,16 +297,15 @@ export class AssembledTransaction<T> {
   }
 
   getStorageExpiration = async () => {
-    const key = new Contract(this.options.contractId).getFootprint()[1]
-
-    const expirationKey = xdr.LedgerKey.expiration(
-      new xdr.LedgerKeyExpiration({ keyHash: hash(key.toXDR()) }),
+    const entryRes = await this.server.getLedgerEntries(
+      new Contract(this.options.contractId).getFootprint()
```

Comment:
keep in mind that this doesn't include the wasm code expiration

Reply:
We previously used the persistent ttl specifically. How would I get that now? How would I get instance ttl?

Reply:
this will give you the instance, just not the wasm